src/query/service/tests/it/servers/http/http_query_handlers.rs

```diff
@@ -1200,19 +1200,35 @@ async fn test_affect() -> Result<()> {
                 is_globals: vec![false],
             }),
             Some(HttpSessionConf {
-                database: None,
+                database: Some("default".to_string()),
                 keep_server_session_secs: None,
                 settings: Some(BTreeMap::from([
                     ("max_threads".to_string(), "1".to_string()),
                     ("timezone".to_string(), "Asia/Shanghai".to_string()),
                 ])),
             }),
         ),
+        (
+            serde_json::json!({"sql": "unset timezone", "session": {"settings": {"max_threads": "6", "timezone": "Asia/Shanghai"}}}),
+            Some(QueryAffect::ChangeSettings {
+                keys: vec!["timezone".to_string()],
+                values: vec!["UTC".to_string()], /* TODO(liyz): consider to return the complete settings after set or unset */
```

Comment:
>>> consider to return the complete settings after set or unset

that will exactly the same as `HttpSessionConf.settings?`

Reply:
we can check if we really need `QueryAffect::ChangeSettings` for set/unset.
if not, just remove it.
otherwise,  it should be the exact effect of the current set/unset SQL.
so  better add a "is_unset" flag. 
for unset, `values` can still be retained. client can choose to just use `is_unset` or the value.

Reply:
IMO this `ChangeSettings` is not quite functional as it expected, we can remove it gradually.